obfx_modules/content-forms/includes/rest/server.php

```diff
@@ -0,0 +1,124 @@
+<?php
+/**
+ *
+ */
+
+namespace ThemeIsle\ContentForms\Includes\Rest;
+
+/**
+ * Class Server
+ * @package ThemeIsle\ContentForms\Rest
+ */
+class Server extends \WP_Rest_Controller {
+
+	/**
+	 * Initialize the rest functionality.
+	 */
+	public function register_hooks() {
+		add_action( 'rest_api_init', array( $this, 'register_endpoints' ) );
+	}
+
+	/**
+	 * Register endpoints.
+	 */
+	public function register_endpoints() {
+
+		register_rest_route(
+			TI_CONTENT_FORMS_NAMESPACE,
+			'/submit',
+			array(
+				'methods'             => \WP_REST_Server::CREATABLE,
+				'callback'            => array( $this, 'submit_form' ),
+				'args'                => array(
+					'form_type'    => array(
+						'type'        => 'string',
+						'required'    => true,
+						'description' => __( 'What type of form is submitted.', 'textdomain' ),
```

Comment:
We need to make the domain consistent. Instead of `textdomain` you can use the plugin slug for it `'themeisle-companion'`

Reply:
@Soare-Robert-Daniel Thank you for the review, I've changed the textdomain and pushed changes.